sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -113,56 +122,68 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
    * @throws Error if an open receiver is already existing for given sessionId.
    */
   private constructor(
+    messageSession: MessageSession,
     context: ClientEntityContext,
     public receiveMode: "peekLock" | "receiveAndDelete",
     private _sessionOptions: CreateSessionReceiverOptions,
     retryOptions: RetryOptions = {}
   ) {
     throwErrorIfConnectionClosed(context.namespace);
     this._context = context;
+    this._sessionOptions;
     this.entityPath = this._context.entityPath;
     this._retryOptions = retryOptions;
+    this._messageSession = messageSession;
```

Comment:
There is a big mis-mash of assignment patterns going on here, lets simplify

```suggestion
    private _messageSession: MessageSession,
    private _context: ClientEntityContext,
    public receiveMode: "peekLock" | "receiveAndDelete",
    private _sessionOptions: CreateSessionReceiverOptions,
    private _retryOptions: RetryOptions = {}
  ) {
    throwErrorIfConnectionClosed(this._context.namespace);
    this.entityPath = this._context.entityPath;
```